src/emulator/auth/operations.ts

```diff
@@ -3055,6 +3051,23 @@ async function fetchBlockingFunction(
     clearTimeout(timeout);
   }
 
+  assert(res, `BLOCKING_FUNCTION_ERROR_RESPONSE : ((Failed to make request to ${url}.))`);
+  const text = await res.text();
```

Comment:
This may throw too, and I guess is better handled in the try-catch block above. You may want to keep only `ok` and `text` as variables.

Reply:
sg